app/code/Magento/InventoryInStorePickup/composer.json

```diff
@@ -6,6 +6,7 @@
         "magento/framework": "*",
         "magento/module-inventory-in-store-pickup-api": "*",
         "magento/module-inventory-distance-based-source-selection-api": "*",
+        "magento/module-inventory-in-store-pickup-shipping-api": "*",
```

Comment:
It is not looking as proper dependency.

Reply:
It will be refactored not in the scope of this issue